pkg/logs/metrics/metrics.go

```diff
@@ -100,6 +100,25 @@ var (
 	LogsTruncated = expvar.Int{}
 	// TlmTruncatedCount tracks the count of times a log is truncated
 	TlmTruncatedCount = telemetry.NewCounter("logs", "truncated", []string{"service", "source"}, "Count the number of times a log is truncated")
+
+	// TlmLogLineSizes is a distribution of post-framer log line sizes
+	TlmLogLineSizes = telemetry.NewHistogram("logs", "log_line_sizes",
```

Comment:
Great metric. 
This will also be useful to us to help understand what common workloads we expect to see - and can be used to tune our SMP benchmarks to represent more realistic workloads.